src/report/business-reports/aging.scm

```diff
@@ -409,8 +510,8 @@ totals to report currency.")
     (define oldintervalreversed (reverse (make-interval-list to-date)))		
   (reverse (cons dayforcurrent oldintervalreversed)))
 
-(define (aging-renderer report-obj reportname account reverse?)
-
+(define (aging-renderer report-obj reportname account reverse? receivable)
+	
```

Comment:
Small nitpick - please remove the blank spaces from this line (git will complain about "trailing whitespace otherwise"). You can leave a completely blank line if you like, it just shouldn't have spaces in it.